src/python/pants/backend/terraform/goals/check_test.py

```diff
@@ -219,7 +219,7 @@ def test_conflicting_provider_versions(rule_runner: RuleRunner) -> None:
     If a large target glob is used (`::`), we get all the sources
     """
     target_name = "in_folder"
-    versions = ["3.2.1", "3.0.0"]
+    versions = ["3.2.1", "3.2.2"]
```

Comment:
3.0.0 is not available on MacOS ARM, so we now conflict two more recent versions in the test.